src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,111 @@
+package duke.ui;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.io.PrintStream;
+import java.util.Scanner;
+
+/** A class that manages the UI */
+public class Ui {
+    public static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    public static final String MESSAGE_GREETING = "Hello from\n" + LOGO;
+    public static final String MESSAGE_HELP = "How can I help you?";
+    public static final String MESSAGE_ADD_TASK = "Added the following task:\n"
+            + "  %1$s\n"
+            + "You have %2$d task(s) in the list";
+    public static final String MESSAGE_DELETE_TASK = "Deleted the following task:\n"
+            + "  %1$s\n"
+            + "You have %2$d task(s) in the list";
+    public static final String MESSAGE_LIST_TASKS = "Your current task(s):\n"
+            + "%1$s";
+    public static final String MESSAGE_FOUND_TASKS = "Found the following task(s):\n"
+            + "%1$s";
+    public static final String MESSAGE_MARK_TASK = "Marked task %1$d as done";
+    public static final String MESSAGE_LOADING_SUCCESS = "Data loaded";
+    public static final String MESSAGE_LOADING_FAIL = "Failed to load data";
+    public static final String MESSAGE_GOODBYE = "Good bye!";
```

Comment:
Any reason why these are all `public` instead of `private`?

Reply:
Since it's constant, I guess it may be a good idea if other classes can access it. Outside user won't be able to modify it either, so this is to improve the flexibility. Will consider to change it to `private` if it is necessary. Thanks!